### DISCUSSION TITLE
Switching WiFI Network

### DISCUSSION THREAD

**Discussion by mushie4282** [OpenVoiceOS/ovos-buildroot#262] [Q&A] [answered]
Do I need to edit a phal-plugin's configuration to swap wireless network? I'm unable to swap wireless network via `raspi-config`. I'm trying to connect to my laptop's hotspot or phone's hotspot.

When I try to connect using raspi-config, the error message is `There was an error running option S1 Wireless LAN`

**Comment by JarbasAl** [comment, marked as answer]
OVOS use network manager, you can use  `nmcli` to change networks, i think raspi-config tries to use wpa_supplicant and thats why it fails? not sure....

alternatively you can ask the PHAL plugin itself do this for you, by emitting a bus message  (this is what wifi setup does)

https://openvoiceos.github.io/message_spec/PHAL_nm/

**Reply by mushie4282** [reply]
I was able to use nmcli to change networks. Thank you for your timely response and help. I don't know how to mark your comment as the answer.

**Reply by JarbasAl** [reply]
to mark an answer it needs to be under Q&A category, otherwise the option doesnt show up, i changed the category and marked it :+1:
